_commands/basics/cd.md

```diff
@@ -16,7 +16,87 @@ $ cd /home
 
 ### Useful Options / Examples
 
+
 #### `cd -`
```

Comment:
Thanks for the feedback, I updated the page to accommodate this and pushed my changes to the forked repo. I am not sure if I need to submit another pull request so let me know if you cannot see the changes.